Alignment/MillePedeAlignmentAlgorithm/interface/MillePedeFileReader.h

```diff
@@ -108,6 +108,8 @@ class MillePedeFileReader {
     return mpPCLresults(updateDB_, vetoUpdateDB_, Nrec_, exitCode_, exitMessage_, updateBits_);
   }
 
+  const std::map<std::string, std::array<bool, 6>> getResultsHG() const { return fractionExceeded_; }
```

Comment:
```suggestion
  const std::map<std::string, std::array<bool, 6>>& getResultsHG() const { return fractionExceeded_; }
```
Wouldn't it be better from the memory performance perspective to have it passed by reference if it's const either way?

Reply:
Done